synapse/models/person.py

```diff
@@ -390,6 +390,9 @@ def getModelDefs(self):
                     ('place', ('geo:place', {}), {
                         'doc': 'The place associated with this contact.',
                     }),
+                    ('place:name', ('geo:name', {}), {
+                        'doc': 'The reported place name of the contact.',
```

Comment:
sounds a little odd to me, maybe "reported name of the place associated with this contact" maybe?